.docs/Dotnet.md

```diff
@@ -9,44 +9,44 @@ If you are using the CLI from Docker, installing the runtime on the host machine
   - [Windows and macOS](#windows-and-macos)
   - [Linux](#linux)
   - [Docker](#docker)
-- [Verify installation](#Verify-installation-optional)
+- [Verify installation](#verify-installation-optional)
 
 ## Windows and macOS
 
-**Download from the direct links:**
+**Download directly from one of the links:**
 
-- [.NET v7.0 Runtime for **Windows x64**](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-7.0.1-windows-x64-installer)
-- [.NET v7.0 Runtime for **Windows x86**](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-7.0.1-windows-x86-installer)
-- [.NET v7.0 Runtime for **Macs with Apple silicon**](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-7.0.1-macos-arm64-installer)
-- [.NET v7.0 Runtime for **Macs with an Intel processor**](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-7.0.1-macos-x64-installer)
+- [.NET v7.0 Runtime for **Windows x64**](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-7.0.11-windows-x64-installer)
+- [.NET v7.0 Runtime for **Windows x86**](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-desktop-7.0.11-windows-x86-installer)
+- [.NET v7.0 Runtime for **Macs with Apple silicon**](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-7.0.11-macos-arm64-installer)
+- [.NET v7.0 Runtime for **Macs with an Intel processor**](https://dotnet.microsoft.com/download/dotnet/thank-you/runtime-7.0.11-macos-x64-installer)
 
 > **Note**:
-> [How can I tell if my computer is running a 32-bit (x86) or a 64-bit (x64) version of Windows?](https://support.microsoft.com/help/15056/windows-32-64-bit-faq)
+[How can I tell if my computer is running a 32-bit (x86) or a 64-bit (x64) version of Windows?](https://support.microsoft.com/help/15056/windows-32-64-bit-faq)
 
 > **Note**:
-> [Is my Mac using an Intel processor or Apple silicon?](https://support.apple.com/HT211814)
+[Is my Mac using an Intel processor or Apple silicon?](https://support.apple.com/HT211814)
```

Comment:
The `>` should probably stay?

Reply:
Yeah, I'll leave it. It doesn't make any difference though with how the blockquote is displayed.

Reply:
Easier to visually parse this way